docs/conf.py

```diff
@@ -84,10 +84,17 @@
 
 
 html_logo = "../logo.png"
+
+#TODO: The option 'logo' is to be removed once pydata-sphinx-theme 
```

Comment:
Add issue (remove TODO?)

Reply:
I added an issue #187 and kept the TODO here.